commcare_connect/opportunity/views.py

```diff
@@ -685,11 +687,22 @@ def user_visits_list(request, org_slug=None, opp_id=None, pk=None):
     opportunity = get_opportunity_or_404(pk=opp_id, org_slug=org_slug)
     opportunity_access = get_object_or_404(OpportunityAccess, pk=pk, opportunity=opportunity)
     user_visits = opportunity_access.uservisit_set.order_by("visit_date")
-    user_visits_table = UserVisitTable(user_visits, org_slug=request.org.slug)
+    visit_filter = UserVisitFilter(request.GET, queryset=user_visits, managed_opportunity=opportunity.managed)
+    user_visits_table = UserVisitTable(
+        visit_filter.qs, org_slug=request.org.slug, template_name="django_tables2/bootstrap5.html"
```

Comment:
Does this queryset include org filter?

Reply:
This table does not filter on orgs. The user visits listed are only for 1 opportunity here so no need for filtering on opportunities or organizations.